### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by Ben-21**

### CODE REVIEW

File: backend/src/main/java/de/neuefische/capstone/backend/imageupload/ImageUploadRepo.java [hunk 1]
```diff
@@ -6,5 +6,4 @@
 
 @Repository
 public interface ImageUploadRepo extends MongoRepository<Image, String> {
-
 }
```
Comment (Pia0912) [inline backend/src/main/java/de/neuefische/capstone/backend/imageupload/ImageUploadRepo.java:9]: I still don't know if it actually makes a difference, but you are missing an empty line here 🙈

File: backend/src/main/java/de/neuefische/capstone/backend/security/MongoUserCreation.java [hunk 2]
```diff
@@ -3,13 +3,11 @@
 import jakarta.validation.constraints.Pattern;
 import jakarta.validation.constraints.Size;
 
-
 public record MongoUserCreation(
         @NotEmpty
-        @Size(min=4, max=14, message = "Username must be between 4 and 14 characters")
+        @Size(min=4, max=30, message = "Username must be between 4 and 14 characters")
```
Comment (Pia0912) [inline backend/src/main/java/de/neuefische/capstone/backend/security/MongoUserCreation.java:8]: also edit your message to display "... and 30 characters"

**Review by Pia0912 — APPROVED** [review]
LGTM! 🚀 Really clean code!

File: backend/src/main/java/de/neuefische/capstone/backend/security/MongoUserCreation.java (hunk 2 above)
Comment (Ben-21) [reply backend/src/main/java/de/neuefische/capstone/backend/security/MongoUserCreation.java:8]: Oha, I am tinkering at the end 😂
Thanks for seeing this!

**Pull request merged**